vnext/Microsoft.ReactNative.Cxx.UnitTests/ReactModuleBuilderMock.h

```diff
@@ -13,7 +13,7 @@
 #include "Windows.UI.Xaml.h"
 
 using namespace winrt;
-using namespace Windows::UI::Xaml;
+using namespace Windows::UI::Xaml; // C++/CX
```

Comment:
> // C++/CX [](start = 34, length = 10)

Why do we need this comment?
This unit test has nothing to do and does not use the C++/CX.
We only use the C++/WinRT.
The previous line adds the `winrt` using.
Thus, this line is really `using namespace winrt::Windows::UI::Xaml;`

Reply:
yep you're right